codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -279,7 +281,7 @@ static int allocateBytesForTree(Http2Stream parent, int connectionWindow) throws
      * A {@link Http2StreamVisitor} that performs the HTTP/2 priority algorithm to distribute the available connection
      * window appropriately to the children of a given stream.
      */
-    private static final class ChildFeeder implements Http2StreamVisitor {
+    private final class ChildFeeder implements Http2StreamVisitor {
```

Comment:
why these can not be static anymore ? I don't see anything in the diff that would have the effect.